docs/LibcxxSupport.md

```diff
@@ -23,30 +29,30 @@ cstddef | Yes | - |
 cstdlib | Partial | **Unsupported functions:** at_quick_exit(), quick_exit() |
 ctime | Partial | All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. <br> **Supported functions:** time(), gettimeofday(), clock_gettime(), nanosleep(). _Please note that clock_gettime() only supports CLOCK_REALTIME_ |
 chrono | Partial | Supported as part of C++11. All time functions implicitly call out to untrusted host for time values. The resulting time values should not be used for security purposes. <br> **Supported classes:** system_clock, treat_as_floating_point, duration_values |
```

Comment:
Updated. I marked those headers that are are affected by C++20 standard as "Supported up to C++17" based. For those introduced by C++11 and unchanged since then, I kept them as "Supported as part of C++11".